GUI/SearchTaskWindow.xaml

```diff
@@ -222,12 +396,62 @@
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
+                                                <LineBreak/>
+                                                This places notches around +/- 1 and 2 Da, such that the issue is accomodated if the difference is slightly off.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept3mm" Content="1, 2, or 3 Missed Monoisotopic Peaks" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
+                                                <LineBreak/>
+                                                This places notches around +/- 1, 2, and 3 Da, such that the issue is accomodated if the difference is slightly off.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept187" Content="-187 and Up" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                Most modifications increase the mass of a peptide, but some decrease it.
+                                                <LineBreak/>
+                                                This is a custom setting for capturing most modifications, where the theoretical mass must be greater than 187 Da less than the experimental.
+                                                <LineBreak/>
+                                                Please use "Modern Search" when using this function.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAcceptOpen" Content="Accept all" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The precursor tolerance is ignored and all theoreticals are searched against all experimentals.
```

Comment:
An "open-mass" search that allows precursor mass-differences of -infinity to infinity. The goal of this search type is to detect mass-differences corresponding to PTMs, amino acid variants, sample handling artifacts, etc. Use "Modern Search" when using this search type.